Content.Server/Cluwne/CluwneSystem.cs

```diff
@@ -0,0 +1,130 @@
+using Content.Server.Administration.Commands;
+using Content.Server.Popups;
+using Content.Shared.Popups;
+using Content.Shared.Mobs;
+using Content.Server.Chat.Systems;
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.Random;
+using Content.Shared.Emoting;
+using Content.Shared.Stunnable;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Damage;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+using Content.Server.Emoting.Systems;
+using Content.Server.Speech.EntitySystems;
+using Content.Shared.Cluwne;
+using Content.Shared.Interaction.Components;
+
+namespace Content.Server.Cluwne;
+
+public sealed class CluwneSystem : EntitySystem
+{
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly SharedStunSystem _stunSystem = default!;
+    [Dependency] private readonly DamageableSystem _damageableSystem = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly ChatSystem _chat = default!;
+    [Dependency] private readonly EmoteSystem _emote = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CluwneComponent, ComponentStartup>(OnComponentStartup);
+        SubscribeLocalEvent<CluwneComponent, MobStateChangedEvent>(OnMobState);
+        SubscribeLocalEvent<CluwneComponent, EmoteEvent>(OnEmote, before:
+            new[] { typeof(VocalSystem), typeof(BodyEmotesSystem) });
+    }
+
+    public override void
+
+     Update(float frameTime)
```

Comment:
Formatting is still screwed here

Reply:
Thanks for giving this a look, i need to sort all formatting issues but currently im waiting for the Add AutoEmote comp/system, updates to zombie code https://github.com/space-wizards/space-station-14/pull/13932. Once this is eventually merged then i can finish off this pr.

Reply:
@brainfood1183 That PR is merged now